src/System.Runtime.Caching/src/Configurations.props

```diff
@@ -6,8 +6,8 @@
       netcoreapp2.0-Windows_NT;
```

Comment:
You can remove the package configurations now and leave the build configurations only.